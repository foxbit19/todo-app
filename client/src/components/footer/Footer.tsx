import React from 'react'
import { Box, Link, Typography } from '@mui/material'
import LoveIcon from '@mui/icons-material/Favorite'

interface Props { }

const Footer = (props: Props) => {
    return (
        <Typography variant='subtitle2' marginTop={10} marginBottom={2} align='center'>
            Made with <LoveIcon fontSize='small' color='primary' /> by <Link href='https://github.com/foxbit19'>Mattia Peretti</Link>.
        </Typography>
    )
}

export default Footer