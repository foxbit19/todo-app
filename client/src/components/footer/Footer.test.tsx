import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('footer', () => {
    test('it matches snapshot', () => {
        const view = render(<Footer />);
        expect(view).toMatchSnapshot();
    });
})