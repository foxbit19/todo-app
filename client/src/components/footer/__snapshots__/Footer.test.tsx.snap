// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer it matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h6
        class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-alignCenter css-1pzwzyj-MuiTypography-root"
      >
        Made with 
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeSmall css-1bptiuh-MuiSvgIcon-root"
          data-testid="FavoriteIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="m12 21.35-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
          />
        </svg>
         by 
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
          href="https://github.com/foxbit19"
        >
          Mattia Peretti
        </a>
        .
      </h6>
    </div>
  </body>,
  "container": <div>
    <h6
      class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-alignCenter css-1pzwzyj-MuiTypography-root"
    >
      Made with 
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeSmall css-1bptiuh-MuiSvgIcon-root"
        data-testid="FavoriteIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="m12 21.35-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
        />
      </svg>
       by 
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
        href="https://github.com/foxbit19"
      >
        Mattia Peretti
      </a>
      .
    </h6>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
